Tests.Integration.Presentation.Web/Tools/External/ItSystemUsageV2Helper.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Presentation.Web.Models.API.V2.Response.SystemUsage;
+using Xunit;
+using Xunit.Sdk;
+
+namespace Tests.Integration.Presentation.Web.Tools.External
+{
+    public static class ItSystemUsageV2Helper
+    {
+        public static async Task<ItSystemUsageResponseDTO> GetSingleAsync(string token, Guid uuid)
+        {
+            using var response = await SendGetSingleAsync(token, uuid);
+            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+
+            return await response.ReadResponseBodyAsAsync<ItSystemUsageResponseDTO>();
+        }
+
+        public static async Task<HttpResponseMessage> SendGetSingleAsync(string token, Guid uuid)
+        {
+            return await HttpApi.GetWithTokenAsync(TestEnvironment.CreateUrl($"api/v2/it-system-usages/{uuid:D}"), token);
+        }
+
+        public static async Task<IEnumerable<ItSystemUsageResponseDTO>> GetManyAsync(
+            string token, 
+            Guid? organizationFilter = null, 
+            Guid? systemUuidFilter = null,
+            Guid? relationToSystemUuidFilter = null,
+            Guid? relationToSystemUsageUuidFilter = null,
+            string systemNameContentFilter = null,
+            int? page = null,
+            int? pageSize = null)
+        {
+            var criteria = new List<KeyValuePair<string, string>>();
+
+            if (organizationFilter.HasValue)
+                criteria.Add(new KeyValuePair<string, string>("organizationUuid", organizationFilter.Value.ToString("N")));
+
+            if (systemUuidFilter.HasValue)
+                criteria.Add(new KeyValuePair<string, string>("systemUuid", systemUuidFilter.Value.ToString("N")));
+
+            if (relationToSystemUuidFilter.HasValue)
+                criteria.Add(new KeyValuePair<string, string>("relatedToSystemUuid", relationToSystemUuidFilter.Value.ToString("N")));
+
+            if (relationToSystemUsageUuidFilter.HasValue)
+                criteria.Add(new KeyValuePair<string, string>("relatedToSystemUsageUuid", relationToSystemUsageUuidFilter.Value.ToString("N")));
```

Comment:
Skifter vi til at vi bruger "N" på Guid? Så skal vi nok også lige rette de andre til

Reply:
i praksis burde deserializer kunne det hele uanset format for det er guid parseren der æder den. Jeg kan bare fjerne det. Så skal man ikke forholde sig til det